src/components/WeatherInfo.js

```diff
@@ -0,0 +1,60 @@
+import Component from "../core/Components.js";
+import { $, convertTimestampToTime } from "../utils.js";
+
+export default class WeatherInfo extends Component {
+  init() {
+    this.state = {
+      currentTime: Math.floor(Date.now() / 1000),
+      weather: [
+        {
+          tempInfo: { average: null, max: null, min: null },
+          weatherInfo: {
+            icon: null,
+            name: null,
+          },
+          name: null,
+        },
+      ],
+    };
```

Comment:
P3;

array의 경우 초기값을 빈 array로 해도 괜찮을 것 같은데 어떠신가요?


```suggestion
    this.state = {
      currentTime: Math.floor(Date.now() / 1000),
      weather: [],
    };
```